pkg/app/impl/authz.go

```diff
@@ -42,8 +43,10 @@ const (
 )
 
 type AuthorizerServer struct {
-	cfg    *config.Common
-	logger *zerolog.Logger
+	cfg      *config.Common
+	logger   *zerolog.Logger
+	issuers  map[string]string
```

Comment:
Should this be a sync.Map as there will be concurrent RW access from various handlers?